src/components/Layout/MainSidebar/SidebarNavItem.tsx

```diff
@@ -1,34 +1,15 @@
 import React from "react";
 import { NavLink as RouteNavLink, useLocation } from "react-router-dom";
-import {
-  NavItem,
-  NavLink,
-  DropdownMenu,
-  DropdownItem,
-  Collapse,
-} from "shards-react";
-
-type SubItem = {
-  title: string;
-  to: string;
-};
-
-type NavigationItem = {
-  title: string;
-  to: string;
-  open: boolean;
-  iconName: string;
-  matches: string[];
-  items?: SubItem[];
-};
+import { NavItem, NavLink } from "shards-react";
+import { TNavItem } from "../../../redux/global/global.types";
 
 type Props = {
-  item: NavigationItem;
+  item: TNavItem;
   toggleSidebar: () => void;
 };
 
 const SidebarNavItem = ({ item, toggleSidebar }: Props) => {
-  const hasSubItems = item.items && item.items.length;
+  const hasSubItems = false;
```

Comment:
This looks like something left from previous changes. Let's drop it. It will simplify attributes of `NavLink` below.

Reply:
Yes, I forgot to remove it :)